pennylane_qrack/qrack_device.py

```diff
@@ -616,6 +616,16 @@ def generate_samples(self):
                 "when using sample-based measurements."
             )
 
+        if self.shots == 1:
+            rev_sample = self._state.measure_all()
+            sample = 0
+            for i in range(self.num_wires):
+                if (rev_sample & (1 << i)) > 0:
```

Comment:
[nit] You don't need outer parens on the if statement:

```py
if rev_sample & (1 << i) > 0:
```

Reply:
You might be right. Here's the Qrack style guide rule-of-thumb though: do you actually remember off the top of your head, definitively, whether bitwise-AND has higher operator precedence that integer greater-than comparison?

No? Then, give it explicit delimiters. Now, anyone at any level of experience can at least trace your order-of-operations by eyeball.